webapp/app/email/client.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import print_function
+
+import json
+
+import sib_api_v3_sdk
+from sib_api_v3_sdk.rest import ApiException
+
+from app.config import Config
+
+
+def add_user_with_doi_and_send_registration_mail(mail):
```

Comment:
Do we need a mocked version for local development?

Reply:
I will have a look into creating a mocked version and add testing usage of it

Reply:
@SannyNguyenHung I added a very simple "mock" that always returns 200 so that we can test that in a cypress test. Further functionality for the mock (email invalid, email already registered and unexpected error) can only be tested through backend tests. Wdyt?